packages/explorer/src/app/api/table/route.ts

```diff
@@ -1,34 +1,34 @@
-import { Hex } from "viem";
-import { KeySchema, ValueSchema } from "@latticexyz/protocol-parser/internal";
 import { getDatabase } from "../utils/getDatabase";
 
 export const dynamic = "force-dynamic";
 
-export type TableConfig = {
-  address: Hex;
-  id: string;
-  key_schema: KeySchema;
-  last_error: string | null;
-  name: string;
-  namespace: string;
-  schema_version: number;
-  table_id: Hex;
-  value_schema: ValueSchema;
+type Row = {
+  [key: string]: string;
 };
 
-export async function GET(req: Request) {
-  const { searchParams } = new URL(req.url);
-  const table = searchParams.get("table") as Hex;
+type Table = Row[] | undefined;
 
-  if (!table) {
-    return Response.json({ error: "table is required" }, { status: 400 });
-  }
+function doesTableExist(tableId: string) {
+  const db = getDatabase();
+  const result = db?.prepare("SELECT name FROM sqlite_master WHERE type='table' AND name = ?").get(tableId);
+
+  return Boolean(result);
+}
+
+export async function GET(request: Request) {
```

Comment:
similar question as above, curious if this could be a server action instead of a separate API route to fetch from